lib/install.go

```diff
@@ -49,6 +49,29 @@ func initialize() {
 
 }
 
+// get install path variable value  (windows os runtime support)
+func getInstallPath() string {
+	return string(os.PathSeparator) + installPath + string(os.PathSeparator)
+}
+
+// get versioned install filename (windows os runtime support)
+func getVersionedInstallFileName(tfversion string) string {
+	if runtime.GOOS == "windows" {
+		return getInstallLocation() + installVersion + tfversion + ".exe"
+	}
+
+	return getInstallLocation() + installVersion + tfversion
+}
+
+// get install filename (windows os runtime support)
+func getInstallFileName() string {
+	if runtime.GOOS == "windows" {
+		return getInstallLocation() + installFile + ".exe"
```

Comment:
```suggestion
		return filepath.Join(getInstallLocation(), installFile + ".exe")
```

Reply:
@Shuliyey looks good.
PertsevRoman  are you able to amend this?